### DISCUSSION TITLE
Unable to connect to Wifi. Homespan installed on DT-ESP32-02 SIP Module (ESP32-PICO-D2 chip)

### DISCUSSION THREAD

**Discussion by rakeshnsw** [HomeSpan/HomeSpan#597] [Q&A] [answered]
I have HomeSpan successfully installed on DT-ESP32-02 SIP Module (pics attached). Status LED is blinking as expected. I am able to enter  into configuration mode but for some reason I am unable to connect to wifi (I tried both CLI and temporary AP methods). Every time HomeSpan just resets to it's original state. Can someone please help me to resolve this issue?

![IMG_1523](https://github.com/HomeSpan/HomeSpan/assets/52046964/489d55a8-9ac4-423f-8125-31da1305d2d7)
![IMG_1525](https://github.com/HomeSpan/HomeSpan/assets/52046964/a5479f6e-2720-45e2-a86c-8116771bdce5)

Here is the detailed log:

`*** COMMAND MODE ***

*** Starting Access Point: HomeSpan-Setup / homespan

Scanning for Networks...

Brownout detector was triggered

ets Jul 29 2019 12:21:46

rst:0xc (SW_CPU_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 188777542, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:1
load:0x3fff0030,len:1184
load:0x40078000,len:13220
ho 0 tail 12 room 4
load:0x40080400,len:3028
entry 0x400805e4

************************************************************
Welcome to HomeSpan!
Apple HomeKit for the Espressif ESP-32 WROOM and Arduino IDE
************************************************************

** Please ensure serial monitor is set to transmit <newlines>

Message Logs:     Level 0
Status LED:       Pin 23
Device Control:   Pin 19
Sketch Version:   n/a
HomeSpan Version: 1.7.2
Arduino-ESP Ver.: 2.0.9
ESP-IDF Version:  4.4.4
ESP32 Chip:       ESP32-PICO-D2 Rev 3 dual-core 4MB FlashESP32 Board:      d1_mini32
PWM Resources:    16 channels, 8 timers, max 20-bit duty resolution
Sodium Version:   1.0.12-idf  Lib 9.4
MbedTLS Version:  mbed TLS 2.28.1
Sketch Compiled:  Jun 18 2023 15:15:56
Partition:        app0
MAC Address:      XX:XX:XX:XX:XX:XX

Device Name:      ESP Switch

*** HomeSpan Info ***

➤ Accessory:  AID=1
   ➟ Service AccessoryInformation:  IID=1, UUID="3E"
      ⇨ Characteristic Name("Switch"):  IID=2, UUID="23", Perms=PR
      ⇨ Characteristic Manufacturer("ESP Switch"):  IID=3, UUID="20", Perms=PR
      ⇨ Characteristic SerialNumber("SW001"):  IID=4, UUID="30", Perms=PR
      ⇨ Characteristic Model("M001"):  IID=5, UUID="21", Perms=PR
      ⇨ Characteristic FirmwareRevision("1.0"):  IID=6, UUID="52", Perms=PR+EV
      ⇨ Characteristic Identify(0):  IID=7, UUID="14", Perms=PW
   ➟ Service HAPProtocolInformation:  IID=8, UUID="A2"
      ⇨ Characteristic Version("1.1.0"):  IID=9, UUID="37", Perms=PR
   ➟ Service LightBulb:  IID=10, UUID="43"
      ⇨ Characteristic On(0):  IID=11, UUID="25", Perms=PR+PW+EV

Configured as Bridge: NO

Database Validation:  Warnings=0, Errors=0

Service                             UUID         AID  IID  Update  Loop  Button  Linked Services
------------------------------  --------  ----------  ---  ------  ----  ------  ---------------
AccessoryInformation                  3E           1    1      NO    NO      NO  -
HAPProtocolInformation                A2           1    8      NO    NO      NO  -
LightBulb                             43           1   10     YES    NO      NO  -

*** End Info ***

Accessory ID:      XX:XX:XX:XX:XX:XX                               LTPK: 7B69DB915A60E9F09C77438DD7D80206DE3E4A13C24BDA2075B9B87DAFFC3B17
No Paired Controllers

Accessory configuration number: 1

*** WIFI CREDENTIALS DATA NOT FOUND.  YOU MAY CONFIGURE BY TYPING 'W <RETURN>'.

ESP Switch is READY!
`

**Comment by HomeSpan** [comment, marked as answer]
Note in your log file it says brownout triggered.  This message is not produced by HomeSpan so much be an ESP-IDF issue.  Do you have anything connected to the ESP32?   If so, try disconnecting completely and re-run.  The brownout message should appear if you have something connected to the ESP32 that is drawing near the maximum power such that when WiFi is activated, the maximum threshold is crossed, the voltage drops, and the ESP32 resets.

**Reply by rakeshnsw** [reply]
I have nothing connected to ESP except just one LED to configure HomeSpan. However I am running this board using FTDI USB adapter. Could that be a reason and not providing enough power?

**Comment by HomeSpan** [comment]
That might be - I've not used an FTID adapter  before.  Also, try without the LED connected.

**Reply by rakeshnsw** [reply]
Connected to external power supply and it is now working perfectly fine without any issues. 😊 Thank You so much for quick resolution.![image](https://github.com/HomeSpan/HomeSpan/assets/52046964/4039b2d3-8c6f-41d6-8eb5-8d51e964f2c6)
